reporter/attrmgr_test.go

```diff
@@ -10,16 +10,14 @@ import (
 	profiles "go.opentelemetry.io/proto/otlp/profiles/v1experimental"
 )
 
-func TestGetSampleAttributes(t *testing.T) {
+func TestAttrTableManager(t *testing.T) {
 	tests := map[string]struct {
 		profile                *profiles.Profile
 		k                      []traceAndMetaKey
-		attributeMap           map[string]uint64
 		expectedIndices        [][]uint64
 		expectedAttributeTable []*common.KeyValue
 	}{
 		"empty": {
-			profile: &profiles.Profile{},
```

Comment:
Not sure why this was in the struct: it seems that simply making this a local temporary variable in the loop below is easier. I reworked it accordingly, but perhaps I'm missing something on why this was done like this originally.